docs/release_notes.rst

```diff
@@ -37,33 +37,76 @@ repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
 
 What's New in 1.2.0
-~~~~~~~~~~~~~~~~~~~
+-------------------
 
-*RTI Connector* 1.2.0 is built on `RTI Connext DDS 6.1.1 <https://community.rti.com/documentation/rti-connext-dds-611>`__.
+*Connector* 1.2.0 is built on `RTI Connext DDS 6.1.1 <https://community.rti.com/documentation/rti-connext-dds-611>`__.
 
 New Platforms
 ^^^^^^^^^^^^^
 
 RTI has validated that *Connector* can be used on macOS 11 (Big Sur) systems.
 
+
+New API makes it easier to query what version of Connector is being used
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. CON-92 
+
+A new API, :meth:`rticonnextdds_connector.Connector.get_version`, has been added that provides the caller
```

Comment:
@samuelraeburn, did you test this link? (I can't test it because when I build py docs, the API Reference doesn't build because I must be missing stuff on my system.) I could test it for js but not for py.

Reply:
![Peek 2021-11-22 10-13](https://user-images.githubusercontent.com/24991686/142834317-7777c484-f310-40c7-bfaf-59b75a1d4208.gif)